go/spec/verbose.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package spec
+
+import (
+	flag "github.com/juju/gnuflag"
+)
+
+var (
```

Comment:
I believe I said this earlier but I wonder if this should move to go/utils rather then live in spec. It's pretty general.